VultisigApp/VultisigApp/Views/Vault/ChainDetailView.swift

```diff
@@ -2,9 +2,16 @@ import SwiftUI
 
 struct ChainDetailView: View {
     @ObservedObject var group: GroupedChain
-    let vault: Vault
     
-    @State var tokens: [CoinMeta] = []
+    let vault: Vault
+
+    var tokens: [Coin] {
+        return vault.coins
+            .filter { $0.chain == group.chain }
+            .filter { !$0.isNativeToken }
```

Comment:
I think you should remove `.filter{ !$0.isNativeToken}` , we need to show the balance of native token
the balance you see on the top is actually the total balance of all tokens